specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-11-20-preview/hanaonazure.json

```diff
@@ -0,0 +1,894 @@
+          },
+          {
+            "$ref": "#/parameters/ProviderInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted provider instance"
+          },
+          "202": {
+            "description": "Deleting provider instance"
+          },
+          "204": {
+            "description": "No provider instance to delete"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OperationList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of HANA operations"
+        }
+      },
+      "description": "List of HANA operations"
+    },
+    "Operation": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the operation being performed on this particular object. This name should match the action name that appears in RBAC / the event service."
+        },
+        "display": {
+          "$ref": "#/definitions/Display",
+          "description": "Displayed HANA operation information"
+        }
+      },
+      "description": "HANA operation information"
+    },
+    "Display": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly form of the resource provider name. This form is also expected to include the publisher/company responsible. Use Title Casing. Begin with \"Microsoft\" for 1st party services."
+        },
+        "resource": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly form of the resource type related to this action/operation. This form should match the public documentation for the resource provider. Use Title Casing. For examples, refer to the “name” section."
+        },
+        "operation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly name for the operation as shown to the user. This name should be concise (to fit in drop downs), but clear (self-documenting). Use Title Casing and include the entity/resource to which it applies."
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly description for the operation as shown to the user. This description should be thorough, yet concise. It will be used in tool-tips and detailed views."
+        },
+        "origin": {
```

Comment:
The property origin is in wrong position. As per ARM operation definition standard, it should be a parallel to 'display' property. To avoid these kind of mismatches, I would recommend referring the common standard definitions directly. Please check the common standard definitions here: https://github.com/Azure/azure-rest-api-specs/blob/2562a49a60b96b3080f4fed35f2db4cb7d5243d6/specification/common-types/resource-management/v2/types.json#L338